### DISCUSSION TITLE
Logs: "A dashboard with the same uid already exists" should be more specific

### DISCUSSION THREAD

**Discussion by danielhoherd** [grafana/grafana#52009] [General]
I see a lot of errors in my grafana logs saying `error="A dashboard with the same uid already exists"` but there is no indication what the UID is, making it difficult to troubleshoot. When I go into the filesystem and `jq` all my dashboards for their UIDs, I don't see a problem. I could regenerate all my UIDs, but that would break all bookmarks to the dashboards, and may not even solve the problem.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
